Tests/PrivateNearestNeighborSearchTests/CiphertextMatrixTests.swift

```diff
@@ -32,12 +32,14 @@ func randomData<T: ScalarType>(dimensions: MatrixDimensions, modulus: T) -> [[T]
     }
 }
 
-final class CiphertextMatrixTests: XCTestCase {
-    func testEncryptDecryptRoundTrip() throws {
+@Suite("Ciphertext Matrix Tests")
```

Comment:
https://developer.apple.com/documentation/testing/suite(_:_:)#parameters - it looks like the suite name will default to `CiphertextMatrixTests` already. So no need for the custom name.
```suggestion
@Suite
```